setpoint.py

```diff
@@ -223,6 +240,9 @@
                 "subclasses": {
                     "Max_Load_Setpoint": {
```

Comment:
Not sure if this is actually a setpoint. It reads like a ``Limit``.

Reply:
Fixed